node-src/lib/logSerializers.test.ts

```diff
@@ -7,8 +7,8 @@ it('strips off envPairs', () => {
   let err;
   try {
     execSync('some hot garbage', { stdio: 'ignore' });
-  } catch (e) {
-    err = e;
+  } catch (err_) {
+    err = err_;
```

Comment:
```suggestion
  } catch (execError) {
    err = execError;
```

Reply:
Turns out, descriptive errors are already supported but it breaks when you add the `name` field for some reason? Once I removed that, I was getting the errors I expected.